### DISCUSSION TITLE
Add currency after installing

### DISCUSSION THREAD

**Discussion by Masterv98** [inventree/InvenTree#5194] [Q&A]
Hi, I was discussing with our technician as we wanted to add the BRL for exchange. Reading the document we discovered that should be done before installing at the config.yaml at the line 79 and 96. But as our team is already using, is it possible to change the pricing money exchange. On my opinion, it should, at least for admins, change the default currency. How can I add the BRL and change it to default? I did not find on the docs.

**Comment by matmair** [comment]
@Masterv98 can you link us this 'document'?
You just need to change the contents of your config file and restart the server. See docs https://docs.inventree.org/en/latest/start/config/?h=currency#supported-currencies
